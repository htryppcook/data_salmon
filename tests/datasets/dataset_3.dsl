dataset {
    int32 field0 = incremental_range(0,100,1)
    string field1 = value("abcd")
    int32 field2 = value(42)
    int32 field3 = ordered_choice(4,8,15,16,23,42)
    ipv4 field4 = random_range("1.2.3.4", "2.3.4.5")
}
