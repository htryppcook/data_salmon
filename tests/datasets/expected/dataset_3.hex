00000000616263640000002a0000000401f3bfb40102030400000001616263640000002a000000080135921f0102030600000002616263640000002a0000000f01b727270102030800000003616263640000002a0000001001e04ec90102030a00000004616263640000002a0000001701a3e78c0102030c00000005616263640000002a0000002a016ab4520102030e00000006616263640000002a0000000401f63da40102031000000007616263640000002a0000000801e4abb80102030400000008616263640000002a0000000f018762430102030600000009616263640000002a000000100121e62001020308