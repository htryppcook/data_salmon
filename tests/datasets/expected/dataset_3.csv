field0,field1,field2,field3,field4,field5
0,abcd,42,4,1.73.81.230,1.2.3.4
1,abcd,42,8,1.146.80.16,1.2.3.6
2,abcd,42,15,1.73.143.198,1.2.3.8
3,abcd,42,16,1.50.144.172,1.2.3.10
4,abcd,42,23,1.130.69.97,1.2.3.12
5,abcd,42,42,1.77.65.138,1.2.3.14
6,abcd,42,4,1.160.206.173,1.2.3.16
7,abcd,42,8,1.52.148.31,1.2.3.4
8,abcd,42,15,1.39.196.149,1.2.3.6
9,abcd,42,16,1.171.18.160,1.2.3.8
